doc/source/methods/PartialDependenceVariance.ipynb

```diff
@@ -0,0 +1,432 @@
+{
```

Comment:
<ul><li>consists in computing the feature importance. -&gt; computes feature importance</li><li>The second type of explanation consists in computing the feature interaction -&gt; The second type of explanation computes feature interaction (being concise helps!)</li><li>I believe the parameters to <code>explain</code> function should be put into back-ticks?</li><li>The results&nbsp;<code>exp_*</code>&nbsp;is -&gt; The result <code>exp</code> is</li><li>exposes an utility function -&gt; exposes a utility</li><li>the feature are sorted -&gt; the features are sorted</li><li><br></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>